ta/pkcs11/src/pkcs11_helpers.c

```diff
@@ -506,6 +506,94 @@ bool key_type_is_asymm_key(uint32_t id)
 	}
 }
 
+/* For AES keys the KCV is 3 bytes length */
+enum pkcs11_rc aes_get_check_value_size(void *key, uint32_t key_size __unused,
```

Comment:
remove `__unused`

Reply:
My bad, I confused `key_size` and `kcv_size`. You were right, keep the `__unused` attribute.
Sorry.